### PR TITLE
makes luxury shuttle windows unbreakable

### PR DESCRIPTION
**Pull request by boodaliboo**
makes luxury shuttle windows unbreakable so the poor can no longer interfere with the rich achievig point of shuttle
:cl:  
tweak: tweaked  fuck the poor
/:cl:

### CODE REVIEW

**Comment by ToasterBiome** [comment]
rules were meant to be broken

**Comment by GoreDem** [comment]
inb4 you can rcd these ones as well

**Comment by Xoxeyos** [comment]
Which windows? The ones where all the poor people go, or the ones on the walls of the shuttle?

**Review by JamieD1 — APPROVED** [review]

File: _maps/shuttles/emergency_luxury.dmm [hunk 1]
```diff
@@ -221,18 +221,6 @@
 /obj/structure/shuttle/engine/propulsion,
 /turf/open/floor/plating/airless,
 /area/shuttle/escape/luxury)
-"oo" = (
-/obj/machinery/door/poddoor/shutters{
-	id = "ohnopoors"
-	},
-/obj/structure/grille/indestructable,
-/obj/structure/window/plastitanium{
-	desc = "A durable looking window made of an alloy of of plasma and titanium. Appears to have poor people tears smudged all over it.";
```
Comment (adamsong) [inline _maps/shuttles/emergency_luxury.dmm:230]: Where did this description go?

**Review by adamsong — CHANGES_REQUESTED** [review]

**Comment by patpol4** [comment]
You replaced them with a closed transparent turf which means it will have space under it and look shit, I'm all for oppression of the poor but this just makes the window look bad. Would be better to use flags that make it indestructible and un rcd'able, if such exist.

**Pull request closed without merging**